docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.0"
+
+services:
+  rabbitmq:
+    container_name: localrabbitmq
+    image: rabbitmq:3.7-management
```

Comment:
@tkuldeep no, you should not use official image. instead you should say: 
```build: .```
so that local Dockerfile gets built and used.

Another problem is in production (Dockerfile), we are using 3.6, but for development you are using 3.7. Better to build locally and use that image.